README.md

```diff
@@ -248,35 +134,25 @@ Feel free to check the [issues page](../../issues/).
 
 ## ⭐️ Show your support <a name="support"></a>
 
-> Write a message to encourage readers to support your project
-
-If you like this project...
+If you like this project give it a ⭐️.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
-
-I would like to thank...
+I would like to thank Microverse community.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- FAQ (optional) -->
 
 ## ❓ FAQ <a name="faq"></a>
 
-> Add at least 2 questions new developers would ask when they decide to use your project.
-
-- **[Question_1]**
-
-  - [Answer_1]
-
-- **[Question_2]**
+- **[Can I use this project for my website?]**
 
-  - [Answer_2]
+  - [Sure It's MIT license, feel free to use as you like]
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] It appears you do not have a `License` file in your directory which you were trying to link to the license to. It's currently showing an error 404 page :point_down: 
![image](https://github.com/MostafaShahat2021/vet_clinic_database/assets/99715640/189a9826-f849-44fe-a617-b61d108d9b86)

Reply:
Hi @AbayomiOlaoye,
I hope all is well with you 👍 ,
thanks a lot for your efforts in reviewing my code,  and for your suggestion, I'll fix that 😄
Best Regards ✌️